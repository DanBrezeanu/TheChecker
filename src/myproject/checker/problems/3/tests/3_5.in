62
1489 8091 2580 1423 7850 7704 5259 121 8163 5667 1199 886 6968 1630 9778 522 5939 9956 2702 5079 6514 8881 4033 7007 4448 63 4388 2397 2020 7325 9951 7075 3118 8566 1109 2827 7973 6842 7276 9332 9720 1577 6476 4011 8868 6712 206 2346 3035 2555 5482 154 879 4404 6099 8374 6467 9127 5005 102 3508 5742 
