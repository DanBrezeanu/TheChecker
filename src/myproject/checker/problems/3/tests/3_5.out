102 154 206 522 886 1630 2020 2346 2580 2702 3118 3508 4388 4404 4448 5482 5742 6476 6514 6712 6842 6968 7276 7704 7850 8374 8566 8868 9332 9720 9778 9956 
9951 9127 8881 8163 8091 7973 7325 7075 7007 6467 6099 5939 5667 5259 5079 5005 4033 4011 3035 2827 2555 2397 1577 1489 1423 1199 1109 879 121 63 
